pkg/config/config.go

```diff
@@ -202,6 +202,7 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("tracemalloc_whitelist", "") // deprecated
 	config.BindEnvAndSetDefault("tracemalloc_blacklist", "") // deprecated
 	config.BindEnvAndSetDefault("run_path", defaultRunPath)
+	config.BindEnvAndSetDefault("no_proxy_nonexact_match", false)
```

Comment:
Let's add it to the config YAML template

Reply:
right